bots/tests-scan

```diff
@@ -33,8 +33,8 @@ DEFAULT_VERIFY = {
     'verify/debian-testing': [ 'master' ],
     'verify/fedora-25': [ ],
     'verify/fedora-i386': BRANCHES,
-    'verify/fedora-26': [ 'master' ],
-    'verify/fedora-27': [ ],
+    'verify/fedora-26': [ ],
```

Comment:
Did you deliberately drop fedora-26 already? If we do this together with PR #7726 it would be consistent, I'm just not sure if it's already the time to stop releasing into F26.

fedora-i386 is F25, but fedora-testing is still F26, so we would at least not completely stop testing Fedora 26.

Reply:
i didn't want to increase the # of tests running on each PR. But we can do it later if you think that is better.

Reply:
Fine for me, I just wanted to make sure it was deliberate.

For this PR I manually triggered a fedora-27 run to verify it.